ios/ReactNativeElementTargeting.swift

```diff
@@ -152,3 +196,82 @@ extension UIView {
             displayName: selector?.displayName)
     }
 }
+
+@available(iOS 13.0, *)
+private extension WKWebView {
+    func children(positionAdjustment: CGPoint) async -> [AppcuesViewElement] {
+        let script = """
+        [...document.querySelectorAll('[id], [data-appcues-id]')].reduce((result, el) => {
+            const { x, y, width, height } = el.getBoundingClientRect();
+            const tag = el.id ? `#${el.id}` : null;
+            const appcuesID = el.getAttribute('data-appcues-id')
+            if (height !== 0 && width !== 0) {
+                result.push({
+                    x,
+                    y,
+                    width,
+                    height,
+                    tag,
+                    appcuesID
+                });
+            }
+            return result;
+        }, []);
+        """
+
+        let response = try? await self.evaluateJavaScript(script)
+
+        guard let objects = response as? [Dictionary<String, Any>] else { return [] }
+
+        return objects.compactMap { element in
+            guard let x = element["x"] as? CGFloat,
+                  let y = element["y"] as? CGFloat,
+                  let width = element["width"] as? CGFloat,
+                  let height = element["height"] as? CGFloat
+            else {
+                return nil
+            }
+
+            let elementFrame = CGRect(x: x, y: y, width: width, height: height)
+            guard self.bounds.contains(CGPoint(x: elementFrame.midX, y: elementFrame.midY)) else {
+                return nil
+            }
```

Comment:
I missed this check in the iOS SDK implementation. Will circle back and add it there.